packages/grpc-js-core/src/call-credentials-filter.ts

```diff
@@ -7,13 +7,27 @@ import {BaseFilter, Filter, FilterFactory} from './filter';
 import {Metadata} from './metadata';
 
 export class CallCredentialsFilter extends BaseFilter implements Filter {
-  constructor(private readonly credentials: CallCredentials) {
+  private serviceUrl: string;
+  constructor(
```

Comment:
Does `CallCredentials` not contain this information? If it does maybe we should prefer using that.

In any case, the call site for this constructor probably needs to be updated.

Reply:
No, the service URL is a property of the call, not the credentials.

And I fixed the constructor and its call site.